cluster/deposit.go

```diff
@@ -45,3 +57,40 @@ func depositDataFromJSON(d depositDataJSON) DepositData {
 		Signature:             d.Signature,
 	}
 }
+
+// VerifyDepositAmounts verifies various conditions about partial deposits rules.
+func VerifyDepositAmounts(amounts []eth2p0.Gwei) error {
```

Comment:
suggest moving these two functions to `eth2util/deposit` package where we have everything related to deposits.

Reply:
Done